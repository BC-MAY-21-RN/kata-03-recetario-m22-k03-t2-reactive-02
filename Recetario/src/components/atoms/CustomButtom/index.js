import React from 'react';
import {Text, Image, StyleSheet, TouchableOpacity} from 'react-native';
import Icon from 'react-native-vector-icons/EvilIcons';

const CustomButtom = props => {
  const {name, onPress} = props;
  return (
    <TouchableOpacity onPress={onPress}>
      <Icon name={name} size={50} color="#f4f5f4" />
    </TouchableOpacity>
  );
};

export default CustomButtom;
