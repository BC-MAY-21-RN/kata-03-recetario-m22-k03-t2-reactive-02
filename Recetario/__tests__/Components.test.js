import 'react-native';
import React from 'react';
import CustomButtom from '../src/components/atoms/CustomButtom';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<CustomButtom />).toJSON();
  expect(tree).toMatchSnapshot();
});
