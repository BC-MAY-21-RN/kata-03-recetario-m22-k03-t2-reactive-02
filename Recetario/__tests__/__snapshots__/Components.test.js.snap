// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <Text
    allowFontScaling={false}
    selectable={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 50,
        },
        undefined,
        Object {
          "fontFamily": "EvilIcons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    
  </Text>
</View>
`;
